blockly-ws-search/workspace_search.js

```diff
@@ -367,12 +370,16 @@ class WorkspaceSearch {
 
   /**
    * Searches the workspace for the current search term.
+   * @param {Blockly.Block=} opt_block Optional block to use as the current
```

Comment:
Looking at code search, it doesn't appear common to use "optional" in description for optional params. Would like to also add note that this is block to use as current if it is included in new search blocks or something like the block to try to use as the current.